proposals/stage-1/TRAVEL_FUND/README.md

```diff
@@ -24,20 +24,19 @@ The role of fund "treasurer" will be removed, as it has been found to be redunda
 
 ## Practical Specifics
 
-- Move relevant files from nodejs/admin to openjs-foundation/cross-project-council such as:
+- Modify these files to account for OpenJS rather than Node.js organizations
```

Comment:
```suggestion
- Modify these files to account for OpenJSF rather than Node.js organizations
```

Reply:
I committed changes, but went with `OpenJS Foundation` instead of `OpenJSF` to stay consistent with the changes suggested by @eemeli earlier in this PR.